### PR TITLE
feat(data-viz): highcharts a11y

### PR DESCRIPTION
**Pull request by krisantrobus**
Added a helper function that will apply any number of Highcharts modules and makes accessibility mandatory. Note, this is only mandatory via variable names as all Highcharts modules have the same type and no namespace to make them distinguishable.

Also did a quick fix for case sensitivity in our component naming matching from airtable.

https://github.com/user-attachments/assets/746de037-7893-4e05-b09f-e23bd11aabfd

### CODE REVIEW

File: packages/paste-website/src/pages/foundations/data-visualization/index.mdx [hunk 1]
```diff
@@ -66,7 +65,20 @@ This foundation page was created to help establish a familiar and accessible use
 
 The easiest way to use the data visualization tokens in your charts is to use `usePasteHighchartsTheme` from the [data visualization library package](/core/libraries/data-visualization) with [Highcharts](https://www.highcharts.com/). The `usePasteHighchartsTheme` hook takes an object of Highchart configurations and returns a new object with Paste colors and fonts.
 
-<LivePreview scope={{Highcharts, HighchartsReact, usePasteHighchartsTheme, LineChartOptions}} language="jsx" noInline>
+We strongly advise that you include the [Highcharts Accessibility module](/foundations/data-visualization#adding-highcharts-accessibility-module) to ensure that your charts are accessible to all UserMessage.
```
Comment (serifluous) [inline packages/paste-website/src/pages/foundations/data-visualization/index.mdx:68]: question: What's UserMessage?

File: packages/paste-website/src/pages/foundations/data-visualization/index.mdx [hunk 2]
```diff
@@ -134,6 +146,26 @@ Shapes can also be used in combination with color-coded data points. Using shape
 
 <LineChartCaption />
 
+### Adding Highcharts Accessibility module
+
+Highcharts by default does no include it's [accessability](https://www.highcharts.com/docs/accessibility/accessibility-module) module. Every Highcarts license includes the Accessibility module which adheres to the WCAG 2.1 standard. You will not need to do anything other than add the module to Highcarts to gain enhanced keyboard navigation and screen reader functionality.
```
Comment (serifluous) [inline packages/paste-website/src/pages/foundations/data-visualization/index.mdx:151]:
```suggestion
Highcharts by default does not include its [Accessibility](https://www.highcharts.com/docs/accessibility/accessibility-module) module. Every Highcharts license includes the Accessibility module which adheres to the WCAG 2.1 standard and adds enhanced keyboard navigation and screen reader functionality.
```

File: packages/paste-website/src/pages/foundations/data-visualization/index.mdx [hunk 3]
```diff
@@ -134,6 +146,26 @@ Shapes can also be used in combination with color-coded data points. Using shape
...
+
+const LineChartExample = () => {
+  return <HighchartsReact highcharts={Highcharts} />;
+};
+```
+
 </content>
 
-</contentwrapper>
+</contentwrapper>applyPasteHighchartsModules,
```
Comment (serifluous) [inline packages/paste-website/src/pages/foundations/data-visualization/index.mdx:171]: issue: loose text here

File: packages/paste-website/src/pages/foundations/data-visualization/index.mdx (hunk 1 above)
Comment (serifluous) [inline packages/paste-website/src/pages/foundations/data-visualization/index.mdx:68]:
```suggestion
Be sure to include the [Highcharts Accessibility module](/foundations/data-visualization#adding-highcharts-accessibility-module) to ensure that your charts are accessible to all UserMessage.
```

Comment (krisantrobus) [reply packages/paste-website/src/pages/foundations/data-visualization/index.mdx:68]: Sorry that's my copilot going crazy chaging user to a variable. fixed in upcoming commit

File: packages/paste-website/src/pages/foundations/data-visualization/index.mdx (hunk 3 above)
Comment (krisantrobus) [reply packages/paste-website/src/pages/foundations/data-visualization/index.mdx:171]: fixed in upcoming commit

File: packages/paste-website/src/components/shortcodes/normalized-component-header/index.tsx [hunk 4]
```diff
@@ -43,7 +43,7 @@ export const NormalizedComponentHeader: React.FC<React.PropsWithChildren<Normali
     githubUrl,
     storybookUrl,
     Feature,
-    shoudShowInPageNav = true,
+    shouldShowInPageNav = true,
```
Comment (nkrantz) [inline packages/paste-website/src/components/shortcodes/normalized-component-header/index.tsx:46]: oops 🙃 nice catch

File: packages/paste-website/src/utils/api.ts [hunk 5]
```diff
@@ -98,7 +98,7 @@ export const getNavigationData = async (): Promise<Feature[]> => {
 export const getFeature = async (feature: string): Promise<Feature> => {
   const data = fs.readFileSync(path.resolve(process.cwd(), "data/feature-data.json"), "utf8");
 
-  return JSON.parse(data).find((item: Feature) => item.Feature === feature);
+  return JSON.parse(data).find((item: Feature) => item.Feature.toLowerCase() === feature.toLowerCase());
```
Comment (nkrantz) [inline packages/paste-website/src/utils/api.ts:101]: 👍

**Review by nkrantz — APPROVED** [review]

**Review by serifluous — APPROVED** [review]

**Pull request merged**